src/nat/front_ends/fastapi/fastapi_front_end_plugin_worker.py

```
@@ -1088,6 +1089,168 @@ async def redirect_uri(request: Request):
+                                "server": "unknown",
+                                "transport": config.server.transport if config.server else "unknown",
+                                "session_healthy": False,
+                                "error": str(e),
+                                "tools": [],
+                                "total_tools": 0,
+                                "workflow_tools": 0
+                            })
+
+                return MCPClientToolListResponse(mcp_clients=mcp_clients_info)
+
+            except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)